src/server/server_family.cc

```diff
@@ -52,6 +53,18 @@ extern "C" {
 
 using namespace std;
 
+struct ReplicaOfFlag {
+  string host;
+  string port;
+
+  operator bool() const {
```

Comment:
Implicit bool operator is evil, see here: https://www.artima.com/articles/the-safe-bool-idiom
Examples:
```cpp
ReplicaOfFlag f;
f << 1;
int i = f;
f < 1;
```
These (and more) will compile, and are probably not what you're after.

You could use the `explicit bool operator`, but I'd personally prefer just `.has_value()` or something.